### PR TITLE
Feature/main menu

### PR DESCRIPTION
**Pull request by duanematyu**
Added the Main Menu
- Start Game
- Quit Game

Difficulty
- Easy
- Normal
- Hard

### CODE REVIEW

File: Assets/Scripts/UI-Scripts/GameManager.DifficultySelection.cs
Comment (Xezner) [inline Assets/Scripts/UI-Scripts/GameManager.DifficultySelection.cs:1]: add Public Class DifficultySelection

File: Assets/Scripts/UI-Scripts/GameManager.MainMenu.cs
Comment (Xezner) [inline Assets/Scripts/UI-Scripts/GameManager.MainMenu.cs:1]: add Public Class MainMenu

change the game objects to properties

File: Assets/Scripts/UI-Scripts/GameManager.DifficultySelection.cs
Comment (duanematyu) [reply Assets/Scripts/UI-Scripts/GameManager.DifficultySelection.cs:1]: added Public Class DifficultySelection

File: Assets/Scripts/UI-Scripts/GameManager.MainMenu.cs
Comment (duanematyu) [reply Assets/Scripts/UI-Scripts/GameManager.MainMenu.cs:1]: added Public Class MainMenu

change the game objects to properties and transferred to the main GameManager class

**Pull request merged**